### ISSUE TITLE
Document the meaning  of values for sys.float_info.rounds

### ISSUE THREAD

**Issue by rhettinger** [python/cpython#85988]
BPO | [41822](https://bugs.python.org/issue41822)
--- | :---
Nosy | @tim-one, @rhettinger, @mdickinson, @serhiy-storchaka, @eamanu, @rahul-kumi

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2020-09-21.00:38:26.708>
labels = ['easy', '3.10', 'docs']
title = 'Document the meaning  of values for sys.float_info.rounds'
updated_at = <Date 2020-10-18.09:13:18.609>
user = 'https://github.com/rhettinger'
```

bugs.python.org fields:
```python
activity = <Date 2020-10-18.09:13:18.609>
actor = 'rhettinger'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2020-09-21.00:38:26.708>
creator = 'rhettinger'
dependencies = []
files = []
hgrepos = []
issue_num = 41822
keywords = ['newcomer friendly']
message_count = 5.0
messages = ['377236', '378852', '378853', '378854', '378856']
nosy_count = 7.0
nosy_names = ['tim.peters', 'rhettinger', 'mark.dickinson', 'docs@python', 'serhiy.storchaka', 'eamanu', 'rahul-kumi']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue41822'
versions = ['Python 3.10']
```

</p></details>

### Linked PRs
* gh-99675

**Comment by tbwolfe** [comment]
PR submitted. Please let me know if any changes are required.

**Comment by mdickinson** [comment]
Closed in #99675.
